src/PublicProfile/profileImage.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+
+const imageContainer = () => {

```

Comment:
could be redundant to have a container just for profileImage. i think you can just add the <img> element in the parent public profile component